services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+    return new_movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
```

Comment:
This method returns QuerySet of MovieSession. Improve the type hint for the return type, please